mergify_engine/engine.py

```diff
@@ -201,37 +201,72 @@ def handle(self, event_type, data):
     # State machine goes here #
     ###########################
 
+    @staticmethod
+    def sort_pulls(pulls):
```

Comment:
at some point we'd just need our own PullRequest class. Can we make such a class and make pygithub use it?

Reply:
Agreed. We currently can't make pygithub returning our object, that why I have done this crappy/lazy monkeypatching stuff. But this crap begin to become a bit fat.